rollup/internal/config/l2.go

```diff
@@ -44,3 +44,9 @@ type BatchProposerConfig struct {
 	GasCostIncreaseMultiplier       float64 `json:"gas_cost_increase_multiplier"`
 	MaxUncompressedBatchBytesSize   uint64  `json:"max_uncompressed_batch_bytes_size"`
 }
+
+// BundleProposerConfig loads bundle_proposer configuration items.
```

Comment:
This struct has not been added to the L2Config struct

Reply:
nice catch, added in [d23b617](https://github.com/scroll-tech/scroll/pull/1411/commits/d23b61720a2e8db8ac78191a1d48ea1b18096188).